### DISCUSSION TITLE
Detail View of Tag field is not clickable

### DISCUSSION THREAD

**Discussion by mrleblanc101** [laravel/nova-issues#4974] [Ideas & Feature Requests]
- Nova Version: 4.17.1

### Description:
First, it think should be a link and not a button.
Second, clicking on the relationship should bring you to the clicked resource, no ?
![Capture d’écran, le 2022-10-26 à 13 37 43](https://user-images.githubusercontent.com/17392251/198097328-ad2876cb-f637-45a9-b780-512104d982ff.png)

**Comment by RhysLees** [comment]
well the fix for this was easy 
```diff
function handleClick() {
  if (props.withPreview) {
    shown.value = !shown.value
+  }else{
+    Nova.visit(`/resources/${props.resourceName}/${props.tag.value}`)
  }
}
```

Anyways, i made it a package, so you can just install it and it will work with the default nova/tag component

https://packagist.org/packages/rhyslees/nova-tags-improved
https://novapackages.com/packages/rhyslees/nova-tags-improved

`will only work on 4.18.x for now untill nova-kit is updated`
